src/font_description.rs

```diff
@@ -12,3 +12,11 @@ impl FromGlibPtrNone<*const ffi::PangoFontDescription> for FontDescription {
         from_glib_none(mut_override(ptr))
     }
 }
+
+impl FontDescription {
+    pub fn from_string(str: &str) -> FontDescription {
+        unsafe {
+            from_glib_full(ffi::pango_font_description_from_string(str.to_glib_none().0))
```

Comment:
This can be done by gir: https://github.com/gtk-rs/gdk/blob/master/Gir.toml#L176-L182